lib/winrm-elevated/runner.rb

```diff
@@ -36,8 +37,8 @@ def initialize(winrm_service)
       # @param [String] The command or PS script to wrap in a scheduled task
       # @param [String] The admin user name to execute the scheduled task as
       # @param [String] The admin user password
-      #
-      # @return [Hash] :stdout and :stderr
+#
+# @return [Hash] :stdout and :stderr
```

Comment:
Looks like this was an accident, which reminds I should probably enable RuboCop on this project and add an automated build.